app/src/components/market/sections/market_create/steps/items/import_market_content.tsx

```diff
@@ -186,15 +186,14 @@ export const ImportMarketContent = (props: Props) => {
 
   const { markets, status: marketIdStatus } = useGraphMarketsFromQuestion(question?.id || '')
   const marketId = markets[0]?.id || ''
-  const { marketMakerData } = useMarketMakerData(marketId.toLowerCase())
+  const { marketMakerData, status } = useMarketMakerData(marketId.toLowerCase())
 
   useEffect(() => {
     if (question && marketIdStatus === Status.Error) {
       setLoading(false)
     } else if (question && marketIdStatus === Status.Ready && !marketMakerData) {
```

Comment:
This is great! Much better already 🤘

I found one flicker of no content when I first load up the page and pop the question in, see recording at around 3s:

https://user-images.githubusercontent.com/72715177/106097718-ebd6bf80-618b-11eb-8499-90431752540f.mov

If you add `status === Status.Ready` instead of `!marketMakerData` at the end of line 194 it seems to get rid of it but I'm not sure if that is the best solution 🤔

Reply:
this oughta fix it